### PR TITLE
Added Intake Sub

### PR DESCRIPTION
**Pull request by Jbrn-2357**
Added Intake subsystem.  Some values within the changeArmPosition method could be incorrect, needs to be tested to find the correct values.

### CODE REVIEW

File: robotbase/src/main/java/com/systemmeltdown/robot/subsystems/IntakeSub.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
+package com.systemmeltdown.robot.subsystems;
+
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+public class IntakeSub extends SubsystemBase {
+
```
Comment (NolanCampbell2357) [inline robotbase/src/main/java/com/systemmeltdown/robot/subsystems/IntakeSub.java:10]: Excess white space.

File: robotbase/src/main/java/com/systemmeltdown/robot/subsystems/IntakeSub.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    private Solenoid m_intakeSolenoid;
+    private WPI_TalonSRX m_intakeTalon;
+    private boolean m_isArmUp = true;
+
+    public IntakeSub(int channel, int intakeTalonID) {
+
```
Comment (NolanCampbell2357) [inline robotbase/src/main/java/com/systemmeltdown/robot/subsystems/IntakeSub.java:16]: Excess white space.

File: robotbase/src/main/java/com/systemmeltdown/robot/subsystems/IntakeSub.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        m_intakeSolenoid = new Solenoid(channel);
+        m_intakeTalon = new WPI_TalonSRX(intakeTalonID);
+
```
Comment (NolanCampbell2357) [inline robotbase/src/main/java/com/systemmeltdown/robot/subsystems/IntakeSub.java:19]: Excess white space.

File: robotbase/src/main/java/com/systemmeltdown/robot/subsystems/IntakeSub.java [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    }
+
+    //percentPowerOuput defintion: -1 = reverse | 0 = stop | 1 = foward
+    public void triggerIntakeRoller(double percentPowerOutput) {
+        m_intakeTalon.set(ControlMode.PercentOutput, percentPowerOutput);   
+    }
+
+    //Current values are most likley incorrect, actual values will be figured out through testing.
+    public void changeArmPosition() {
+
```
Comment (NolanCampbell2357) [inline robotbase/src/main/java/com/systemmeltdown/robot/subsystems/IntakeSub.java:29]: Excess white space.

File: robotbase/src/main/java/com/systemmeltdown/robot/subsystems/IntakeSub.java [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+        if(m_isArmUp) {
+            m_intakeSolenoid.set(false);
+            m_isArmUp = false;
+
+        } else {
+            m_intakeSolenoid.set(true);
+            m_isArmUp = true;
+        }
+
```
Comment (NolanCampbell2357) [inline robotbase/src/main/java/com/systemmeltdown/robot/subsystems/IntakeSub.java:38]: Excess white space.

**Review by NolanCampbell2357 — CHANGES_REQUESTED** [review]

**Review by GameMagma — CHANGES_REQUESTED** [review]
Delete the excess lines (line 10, line 16, line 19, line 29, and line 38) and then it's fine.

**Pull request merged**